odoo_module_migrate/migration.py

```diff
@@ -170,3 +171,13 @@ def run(self):
                 self._directory_path.resolve()))
         for module_migration in self._module_migrations:
             module_migration.run()
+
+        if os.path.exists(
+                os.path.join(self._directory_path, '.pre-commit-config.yaml')
+        ):
+            logger.info('File .pre-commit-config.yaml finded. '
+                        'Running pre-commit')
+            subprocess.run(
+                ["pre-commit", "run", "-a"],
```

Comment:
This runs pre-commit for all files, while it should be for modules specified in ``--modules``